task33.py

```diff
@@ -0,0 +1,22 @@
+import threading
+
+
+counter = 0
+rounds = 100000
+
+class Counter(threading.Thread):
+    def run(self):
+        global counter
+        for _ in range(rounds):
+            counter += 1
```

Comment:
You should have used locks here, otherwise it might not work correctly

Reply:
Or at least provide an explanation of why it works correctly on your machine (google it)